.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,6 +4,10 @@ Thank you for your contribution!
 Please review https://github.com/ray-project/ray/blob/master/CONTRIBUTING.rst before opening a pull request.
 -->
 
+## Why are these changes needed?
+
+<!-- Please give a short brief about the problem these changes address. -->
```

Comment:
```suggestion
<!-- Please give a short summary of the problem these changes address. -->
```

This is not directly related to your change, but while we're at it might be good to change this wording. "Brief" as a noun is not a very common word so it could be less understandable (especially to non-native English speakers).

Also, "short brief" is pretty redundant...

Reply:
Applies to the "what" section below as well